PHPCI/Plugin/Email.php

```diff
@@ -70,12 +70,21 @@ public function execute()
 
         $buildStatus  = $this->build->isSuccessful() ? "Passing Build" : "Failing Build";
         $projectName  = $this->build->getProject()->getTitle();
-        $mailTemplate = $this->build->isSuccessful() ? 'Email/success' : 'Email/failed';
+        $mailTemplate = $this->build->isSuccessful() ? 'Email/long' : 'Email/short';
```

Comment:
You're right and I agree. I just have kept the current behavior. Should we change that ?